autogen/agentchat/groupchat.py

```diff
@@ -232,17 +257,22 @@ def select_speaker_msg(self, agents: Optional[List[Agent]] = None) -> str:
         """Return the system message for selecting the next speaker. This is always the *first* message in the context."""
         if agents is None:
             agents = self.agents
-        return f"""You are in a role play game. The following roles are available:
-{self._participant_roles(agents)}.
 
-Read the following conversation.
-Then select the next role from {[agent.name for agent in agents]} to play. Only return the role."""
+        roles = self._participant_roles(agents)
+        agentlist = f"{[agent.name for agent in agents]}"
+
+        return_msg = self.select_speaker_message_template.format(**locals())
```

Comment:
I think it would be better to specify the specific values you are replacing rather than substituting the whole local environment

Reply:
Fair point, let me update

Reply:
Okay, all done :)